### ISSUE TITLE
Getting a WDT Reset Loop

### ISSUE THREAD

**Issue by Cellgalvano** [walmis/blackmagic-espidf#10]
Hi!

I tried the BMP using a prebuilt binary I found online and it worked great. Currently I'm trying to build the newest version with Live Expressions.

I can build the firmware and flash it to a NodeMCU (ESP8266-12E) and I always get into a boot loop caused by a watchdog reset. 

On the UART I get:
rst cause: 4, boot mode:(3,6)
wdt reset
load 0x60000014, len 8, room 16
tail 8

As a build system I'm using a fresh Ubuntu 20.04.2 VM with the xtensa toolchain version 8.4.0 (2020r3).

Hope someone can help.
Best regards!

**Comment by Cellgalvano** [comment]
I finally got it to work on multiple ESP8266-12E modules!

In my case I needed to do a "erase_flash" using esptool.py before running "make flash". The modules were used in another application and the flash held some kind of unrelated data.

Before I would close this issue I would life to know if it would be an improvement to change the "make flash" command to erase the whole flash before writing the image?

Best regards!
